tools/src/main/java/org/apache/kafka/trogdor/workload/ConsumeBenchWorker.java

```diff
@@ -270,9 +270,9 @@ public Void call() throws Exception {
                 long curTimeMs = Time.SYSTEM.milliseconds();
                 log.info("{} Consumed total number of messages={}, bytes={} in {} ms.  status: {}",
                          clientId, messagesConsumed, bytesConsumed, curTimeMs - startTimeMs, statusData);
+                consumer.close();
```

Comment:
It seems like we'd want to close the consumer before completing `doneFuture` right? Given that, it seems like we should do it before calling `abort` and right after the `finally`. What do you think?

Reply:
Also, I suggest renaming `abort` to `abortAndThrow`.

Reply:
Agreed, we're not done until we close the consumer